Test.Microsoft.NET.Build.Containers.Filesystem/DockerRegistryManager.cs

```diff
@@ -34,26 +31,30 @@ public static void StartAndPopulateDockerRegistry(TestContext context)
         Assert.IsNotNull(registryContainerId);
         registryProcess.WaitForExit();
         Assert.AreEqual(0, registryProcess.ExitCode, $"Could not start Docker registry. Are you running one for manual testing?{Environment.NewLine}{errStream}");
-
         s_registryContainerId = registryContainerId;
 
+
+        context.WriteLine("Pulling base image");
         Process pullBase = Process.Start("docker", $"pull {BaseImageSource}{BaseImage}:{BaseImageTag}");
         Assert.IsNotNull(pullBase);
         pullBase.WaitForExit();
         Assert.AreEqual(0, pullBase.ExitCode);
-
+        
+        context.WriteLine("Tagging base image");
         Process tag = Process.Start("docker", $"tag {BaseImageSource}{BaseImage}:{BaseImageTag} {LocalRegistry}/{BaseImage}:{BaseImageTag}");
         Assert.IsNotNull(tag);
         tag.WaitForExit();
         Assert.AreEqual(0, tag.ExitCode);
 
+        context.WriteLine("Pushing base image to local registry");
         Process pushBase = Process.Start("docker", $"push {LocalRegistry}/{BaseImage}:{BaseImageTag}");
         Assert.IsNotNull(pushBase);
         pushBase.WaitForExit();
         Assert.AreEqual(0, pushBase.ExitCode);
+
+        context.WriteLine("Continuing with the tests");
```

Comment:
```suggestion
        context.WriteLine($"{nameof(StartAndPopulateDockerRegistry)} complete");
```

? Since it's not the last thing?